client/my-sites/patterns/components/page-view-tracker.tsx

```diff
@@ -52,7 +52,7 @@ export function PatternsPageViewTracker( {
 				category,
 				is_logged_in: isLoggedIn,
 				user_is_dev_account: isDevAccount ? '1' : '0',
-				search_term: searchTerm,
+				search_term: searchTerm ? searchTerm : undefined,
```

Comment:
```suggestion
				search_term: searchTerm || undefined,
```
Nit. Suggested this handy shorthand also used in other places: https://github.com/Automattic/wp-calypso/blob/7969cbed62854e57b9162217844d59a13e14429a/client/my-sites/patterns/hooks/use-record-patterns-event.tsx#L14

Reply:
Yeah, totally agree, my bad